specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-05-30-preview/share.json

```diff
@@ -212,7 +212,7 @@
             "$ref": "#/parameters/filter"
           },
           {
-            "$ref": "#/parameters/orderby"
+            "$ref": "#/parameters/orderBy"
```

Comment:
Why make this change? Unless there is some strong reason for it I think it would be better to leave the name as is.

Reply:
Suggestion from the SDK review.

Reply:
It should be renamed in the SDKs -- not in the API. Specifically I would recommend adding a transform for .NET and Java if these are the languages requesting the new name.  Python and JavaScript typically choose to use names from the API, but you could verify with the architects.

Reply:
Yes it can be done, that was my thoughts. We have done something similar before as well.

Reply:
> It should be renamed in the SDKs -- not in the API. Specifically I would recommend adding a transform for .NET and Java if these are the languages requesting the new name. Python and JavaScript typically choose to use names from the API, but you could verify with the architects.

Java codegen already do the transform automatically
```java
    ODATA_PARAMETER_NAME_CONVERSION.put("maxpagesize", "maxPageSize");
    ODATA_PARAMETER_NAME_CONVERSION.put("orderby", "orderBy");
```